### PR TITLE
CSI-1305_readme_allignment

### PR DESCRIPTION
**Pull request by igorgonibm**
Align operator and driver readme files and yaml examples

Parallel operator review: https://github.com/IBM/ibm-block-csi-operator/pull/74

### CODE REVIEW

**Comment by oriyarde** [comment]
@igorgonibm according to the ticket description, I assume this should fix the remaining conversations in https://github.com/IBM/ibm-block-csi-driver/pull/144.
is so, please fix them here aswell.
if not, then do we have another ticket for it?

File: README.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ This is the official operator to deploy and manage IBM block storage CSI driver.
 Supported container platforms:
   - OpenShift v4.2
   - OpenShift v4.3
-  - Kubernetes v1.14
   - Kubernetes v1.16
+  - Kubernetes v1.17
```
[the remark this answers is not in the document]
Comment (igorgonibm) [reply README.md:10]: Adding it is not part of the ticket
This change was not part of it too but it was easy to add so i did.

[the remark this answers is not in the document]
Comment (oriyarde) [reply README.md:10]: I think that we should update both kubernetes and openshift in the same ticket.
not sure why kubernetes is easier than openshift.
if we plan to update them in different tickets, do we have a ticket for the openshift updates?

[the remark this answers is not in the document]
Comment (igorgonibm) [reply README.md:10]: Updated and also in operator

File: README.md [hunk 2]
```diff
@@ -106,49 +242,66 @@ Create a secret file as follows `array-secret.yaml` and update the relevant cred
...
+
+To create the secret using a command line terminal, use the following command:
+```bash
+kubectl create secret generic <NAME> --from-literal=username=<USER> --fromliteral=password=<PASSWORD> --from-literal=management_address=<ARRAY_MGMT> -n <namespace>
 ```
 
 #### 2. Create storage classes
 
-Create a storage class yaml file `storageclass-gold-svc.yaml` as follows, with the relevant capabilities, pool and, array secret:
+Create a storage class `demo-storageclass-gold-pvc.yaml` file as follows, with the relevant capabilities, pool and, array secret.
```
Comment (oriyarde) [inline README.md:268]: why is it `demo-` if the values have template placeholders?

File: README.md [hunk 3]
```diff
@@ -63,41 +131,109 @@ systemctl status multipathd
...
+$ kubectl create ns <namespace>
+```
+
+#### 4. Install the operator, while using a user-defined namespace.
+
+```bash
+$ kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+```
+
+### Verify the operator is running:
```
Comment (oriyarde) [inline README.md:185]: @ArbelNathan I think this `###` should be something like `#####` - what do you think? and should we fix it soon?

**Pull request merged**